src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Assertion.cs

```diff
@@ -84,6 +89,46 @@ public Saml2Conditions Conditions
             set;
         }
 
+        /// <summary>
+        /// Gets the canonicalized (ExclusiveC14n) representation without comments.
+        /// </summary>
+        public string CanonicalString
```

Comment:
Would it make sense to convert this property to a method, allowing users to specify includeComments and inclusiveNamespacesPrefixList arguments?

Reply:
Same as above.